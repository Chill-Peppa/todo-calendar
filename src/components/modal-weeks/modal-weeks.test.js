import renderer from 'react-test-renderer';
import ModalWeeks from './modal-weeks';

it('should correctly render ModalOverlay', () => {
  const tree = renderer.create(<ModalWeeks />).toJSON();
  expect(tree).toMatchSnapshot();
});
