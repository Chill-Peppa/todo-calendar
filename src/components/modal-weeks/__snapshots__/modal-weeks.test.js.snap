// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ModalOverlay 1`] = `
<div
  className="modal-weeks"
>
  <div
    className="modal-week__inputs"
  >
    <div
      className="modal-week__input-container"
    >
      <label
        htmlFor="start"
      >
        При выборе любого дня отобразится список задач на эту неделю
      </label>
      <input
        id="start"
        name="trip-start"
        onChange={[Function]}
        type="date"
        value=""
      />
    </div>
  </div>
  <p
    className="modal-weeks__without-tasks"
  >
    На этой неделе планов нет.
  </p>
</div>
`;
